File: Modul-6/Soal-5/PRAK602-2410817110016-MuhammadAzriRahman.c
#include <stdio.h>

void main() {
    int n;

    printf("Input\n");
    scanf("%d", &n);

    int matriksA[n][n];
    int matriksB[n][n];
    int matriksHasil[n][n];

    printf("Matriks A\n");
    for (int i = 0; i < n; i++) {
        for (int j =0; j < n; j++) {
            scanf("%d", &matriksA[i][j]);
        }
    }
    printf("Matriks B\n");
    for (int i = 0; i < n; i++) {
        for (int j =0; j < n; j++) {
            scanf("%d", &matriksB[i][j]);
        }
    }
    printf("Output\n");
    printf("Matriks A*B\n");
    for (int i = 0; i < n; i++) {
        for (int j =0; j < n; j++) {
            matriksHasil[i][j] = 0;
            for (int k = 0; k < n; k++) {
                matriksHasil[i][j] += matriksA[i][k] * matriksB[k][j];
            }
        }
    }
    for (int i = 0; i < n; i++) {
        for (int j = 0; j < n; j++) {
            printf("%d ", matriksHasil[i][j]);
        }
        printf("\n");
    }
}

File: Modul-6/Soal-5/PRAK602-2410817110016-MuhammadAzriRahman.py
print("Input")
n = int(input(""))

print("Matriks A")
MatriksA = []
for i in range(n):  
    baris = list(map(int, input().split())) 
    MatriksA.append(baris)

print("Matriks B")
MatriksB = []
for i in range(n):
    baris = list(map(int, input().split())) 
    MatriksB.append(baris)

MatriksHasil = [[0] * n for _ in range(n)]
for i in range(n):  
    for j in range(n): 
        for k in range(n):  
            MatriksHasil[i][j] += MatriksA[i][k] * MatriksB[k][j]

print("Output")
print("Matriks A*B")
for baris in MatriksHasil: 
    print(" ".join(map(str, baris))) 
